packages/alias/rollup.config.js

```diff
@@ -1,10 +1,13 @@
+import typescript from '@rollup/plugin-typescript';
+
 import pkg from './package.json';
 
 export default {
-  input: 'src/index.js',
-  external: [...Object.keys(pkg.dependencies), 'path', 'fs', 'os'],
+  input: 'src/index.ts',
+  external: [...Object.keys(pkg.dependencies), 'os'],
```

Comment:
This allows to not modify externals at all
```
external: id => !id.startsWith('.') && !path.isAbsolute(id)
```

Reply:
I'm happy to do this but we should be consistent, all other packages currently use a hardcoded list. We could also use [`builtinModules`](https://nodejs.org/api/modules.html#modules_module_builtinmodules).